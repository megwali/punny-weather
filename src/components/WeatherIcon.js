
const WeatherIcon = ({ icon, weatherCondition}) => (
  <img
    alt={`${weatherCondition}-weather-icon`}
    height="120"
    src={`${process.env.REACT_APP_ICON_URL}${icon}@2x.png`}
    width="120"
  />
);

export default WeatherIcon;
