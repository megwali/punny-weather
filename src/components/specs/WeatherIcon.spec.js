import { create } from 'react-test-renderer';
import WeatherIcon from '../WeatherIcon';

describe('WeatherIcon', () => {
  it ('renders', () => {
    const component = create(<WeatherIcon icon="test" weatherCondition="test" />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
