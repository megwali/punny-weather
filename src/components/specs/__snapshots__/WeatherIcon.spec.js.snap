// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WeatherIcon renders 1`] = `
<img
  alt="test-weather-icon"
  height="120"
  src="https://openweathermap.org/img/wn/test@2x.png"
  width="120"
/>
`;
